io.github.darkhz.rclonetui.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.github.darkhz.rclonetui</id>
+  
+  <name>rclone-tui</name>
+  <summary>A TUI based manager for rclone</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      rclone-tui is a cross-platform manager for rclone. It aims to be on-par with the web GUI  as well as provide more improvements and enhancements over its general functionality.
+    </p>
+
+    <p>
+    Its features currently include: 
+    - Monitor rclone stats via the dashboard 
+    - Create, update, view and delete configurations
+    - Explore remotes and perform various operations
+    - Mount and unmount remotes 
+    - View file transfer and progress information
+    </p>
+
+    <p>Current version: v0.0.2</p>
+  </description>
+  
+  <icon type="stock">utilities-terminal</icon>
```

Comment:
stock icon should not be used.

Also missing: `<releases>`, `<url>`, `<screenshots>` and `<content_rating>`.